File: jmlparser-jml-tools/src/main/java/com/github/jmlparser/pp/PrettyPrintCommand.java
package com.github.jmlparser.pp;

import com.beust.jcommander.Parameter;
import com.beust.jcommander.Parameters;

import java.util.ArrayList;
import java.util.List;

/**
 * @author Alexander Weigl
 * @version 1 (09.04.23)
 */
@Parameters(commandNames = {"format"},
        commandDescription = "Format the JML comments inside Java files.")
public class PrettyPrintCommand {
    @Parameter(description = "Files to check", required = true)
    private List<String> files = new ArrayList<>();

    public void run() {
    }
}

File: tools/cli/src/main/kotlin/io/github/jmltoolkit/cli/PrettyPrintCommand.kt
package io.github.jmltoolkit.cli

import com.github.ajalt.clikt.core.CliktCommand
import com.github.ajalt.clikt.core.Context
import com.github.ajalt.clikt.parameters.arguments.argument
import com.github.ajalt.clikt.parameters.arguments.multiple
import com.github.ajalt.clikt.parameters.types.file

/**
 * @author Alexander Weigl
 * @version 1 (09.04.23)
 */
class PrettyPrintCommand : CliktCommand(name = "format") {
    override fun help(context: Context): String = "Format the JML comments inside Java files."

    private val files by argument("FILES").file().multiple()
    override fun run() {}
}
